sidebar/Rsidebar/housing_stock_data_municipalities.R

```diff
@@ -0,0 +1,37 @@
+#' Read housing stock data
+#'
+#' This function retrieves housing stock counts for 2015-2019 of some
+#' municipalities from provinces of Groningen and Drenthe
+#' @importFrom jsonlite read_json
+#' @noRd
+read_housing_stock_data2 <- function() {
+  data = read.csv(file = "C:/Users/Augustin/Desktop/housingmarket_municip_Gr_DR.csv", head = TRUE, sep = ";")
+}
```

Comment:
Is this the excel file linked in https://github.com/ikbentimkramer/cdphmd/issues/24#issuecomment-721922649?